x-pack/plugins/uptime/public/contexts/index.ts

```diff
@@ -13,3 +13,4 @@ export {
   UptimeStartupPluginsContext,
   UptimeStartupPluginsContextProvider,
 } from './uptime_startup_plugins_context';
+export { UptimeCheckStepsContextProvider } from './uptime_check_steps_context';
```

Comment:
I'm getting errors that this file can't be found; I can't seem to find it either, did something weird happen on a rebase or am I missing something? Running `9bd2662fd39896f83105f047dd8e7926ce02c2c9` commit.

Reply:
@justinkambic yes! I had just ran the command to push this file this but apparently the command failed and it wasn't there. It should be okay on the next build now.